### ISSUE TITLE
Bibliography options

### ISSUE THREAD

**Issue by kxxdhdn** [ElegantLaTeX/ElegantBook#112]
- ElegantBook v.4.0.2 via GitHub (elegantbook.cls file)
- Platform : Emacs v26.3 (AUCTEX v12.2.4) on macOS Catalina v10.15.6

### Description : 
I got some troubles when I tried to choose citation styles. Basically I wanted to use the natbib styles. It seems that the `cite=numbers/super/authoryear` option proposed in the documentation file does not work. Neither does `natbib=true`. Indeed when I checked the elegantbook.cls file, I could not find a corresponding definition (something like `\DeclareOption[super]{cite}`). 

So I tried to define it in my tex file using `\usepackage[backend=biber,style=alphabetic,citestyle=authoryear]{biblatex}
`. It turned out that the package biblatex had already been used. Then I found the following code in elegantbook.cls file : 

```latex
\ifdefstring{\ELEGANT@lang}{cn}{
  \RequirePackage[
    backend=biber,
    style=gb7714-2015,
    sorting=gb7714-2015]{biblatex}}{
  \RequirePackage[
    backend=biber,
    style=numeric-comp,
    sorting=none]{biblatex}
}
```

After replacing `style=numeric-comp,` by

```latex
style=alphabetic,
citestyle=authoryear,
```

I got what I wanted : `(Author et al. 2011)` with the command `\parencite` or `Author et al. (2011)` with the command `\textcite`. Note that `\documentclass[en,14pt,cyan,fancy,style=alphabetic,citestyle=authoryear]{elegantbook}` does NOT do the same thing. I wonder whether the template itself provides such flexibility.

By the way, I tried also `\usepackage{natbib}` in order to use `\citep` and `\citet` instead of `\textcite` and `\parencite`, but there were "already defined" \bibhang, \NAT@parse and \citename, which may be due to the incompatibility with biblatex package.

P.S. I performed "Clean All - Recompile with XeLaTeX - Recompile with Biber - Recompile with XeLaTeX" every time I modified the options or input packages.

**Comment by sikouhjw** [comment]
4.0.3 is the version without release, so the instruction manual does not say how to enter in `\documentclass`. If you are using elegantBook now, it is recommended to go to https://github.com/ElegantLaTeX/ElegantBook/issues/106

**Comment by kxxdhdn** [comment]
Ok. Problem fixed with version3x branch (frozen 3.11 scripts). I will close it, thanks.

And good luck for biblatex inclusion in version4x.
